package.json

```diff
@@ -15,6 +15,11 @@
   "keywords": [
     "sassdoc-theme"
   ],
+  "main": "./index.js",
+  "exports": {
+    "sass": "./scss/_utilities.scss",
```

Comment:
Should the docs at  https://www.oddbird.net/herman/docs/api_json-export be updated to recommend using `pkg:sassdoc-theme-herman`? It also strikes me as odd to have a utility be the main export, it could be available at `./utilities` or something, but that's also odd because then this doesn't have a default export.

Reply:
> Should the docs at https://www.oddbird.net/herman/docs/api_json-export be updated to recommend using `pkg:sassdoc-theme-herman`?

Hm, that's a good question. It's difficult to use `pkg:` in the actual compiled examples, because I don't think Herman can also have Herman installed in `node_modules`? 🤯 But at the very least we could add a comment documenting the recommended usage -- I'll add that.

> It also strikes me as odd to have a utility be the main export, it could be available at `./utilities` or something, but that's also odd because then this doesn't have a default export.

In this case `scss/_utilities.scss` is really the only file that users should ever be importing, so I think this makes sense? I guess we could rename it though.

Reply:
> I don't think Herman can also have Herman installed

Possible, but that way lies danger :-D. I didn't do much digging on how the docs were built.

I don't think it's worth renaming

Reply:
> Possible, but that way lies danger :-D. I didn't do much digging on how the docs were built.

Yeah, I ran into this same issue in True when I tried to test using `pkg:sass-true` and realized that `sass-true` wasn't a valid installed package. 🤦

Reply:
Added in https://github.com/oddbird/sassdoc-theme-herman/pull/416/commits/81250ed5aad66c92db4a930b2aaceb2c1aa62b82

@mirisuzanne @jamesnw Do you think that's clear enough?

Reply:
I was just thinking through if we should have added some method for self-referencing/internal pkg urls to Sass, but... nah.